sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/rest_client.hpp

```diff
@@ -921,6 +919,17 @@ namespace Azure { namespace Storage { namespace Blobs {
          */
         std::string LeaseId;
       };
+      struct BlobName final
+      {
+        /**
+         * Indicates if the blob name is encoded.
+         */
+        bool Encoded = bool();
+        /**
+         * The name of the blob.
+         */
+        std::string Content;
```

Comment:
If `Content` has the ability to be "encoded", then it shouldn't be represented as a `std::string` (which is either ASCII or UTF-8 text). Instead if `Content` can be encoded, this should probably be `std::vector<uint8_t>`.

Reply:
Ok, reading further, this is URL encoded. Since URL encoded text is ASCII text, using std::string is probably reasonable.